src/components/markdown/Spoiler.tsx

```diff
@@ -1,6 +1,51 @@
-import * as React from 'react';
+import React, { Children } from 'react';
+
+function determineIfSingularCodeBlock(
+  firstName: string,
+  numChildren: number
+): boolean {
+  return firstName == 'LanguageSection' && numChildren == 1;
+}
 
 const Spoiler = ({ children, title }) => {
+  let count = 0;
+  let numChildren = 0;
+  let firstName = 'None';
+  let ogProps;
+
+  React.Children.forEach(children, child => {
+    if (count == 0) {
+      numChildren = Children.count(children);
```

Comment:
You should choose either `React.Children` or `Children` to be consistent.
```suggestion
      numChildren = React.Children.count(children);
```